src/zoned-date-time.js

```diff
@@ -104,4 +104,6 @@ ZonedDateTime.prototype.toDate = function() {
   };
 });
 
-module.exports = ZonedDateTime;
+if (typeof module !== "undefined" && module.exports) {
```

Comment:
Thank you. Could we simplify the condition with this?

```js
```suggestion
if (module !== undefined && module.exports) {
```

Or even this?
```suggestion
if (module && module.exports) {
```

Thanks

Reply:
I don't think we can. The problem we are addressing is that `module` is not a declared global variable normally in browsers. A global variable lookup can only be done safely through `typeof`. Any other check would produce an uncaught ReferenceError exception.

E.g. try on blank page or https://example.org

```js
if (foobar) {} //> Uncaught ReferenceError: foobar is not defined
if (module) {} //> Uncaught ReferenceError: module is not defined
console.log(module); //> Uncaught ReferenceError: module is not defined
```

The reason this worked previously on the Globalize tests, is that there happened to be a legacy global `function module()` defined by QUnit 1.x which is unrelated to CommonJS but happened to make this work.

Other examples of conditional CJS exports e.g. from Rollup or UMD tend to work the same way. [[1]](https://github.com/rollup/plugins/blob/commonjs-v16.0.0/packages/commonjs/test/fixtures/samples/define-is-undefined/foo.js) [[2]](https://github.com/rollup/plugins/blob/commonjs-v16.0.0/packages/commonjs/test/fixtures/samples/umd/sinon.js)

Reply:
True! Thanks